python/pyspark/sql/readwriter.py

```diff
@@ -828,8 +828,7 @@ def csv(self, path, mode=None, compression=None, sep=None, quote=None, escape=No
                     set, it uses the default value, ``,``.
         :param quote: sets the single character used for escaping quoted values where the
                       separator can be part of the value. If None is set, it uses the default
-                      value, ``"``. If you would like to turn off quotations, you need to set an
-                      empty string.
+                      value, ``"``. If empty string is set, it uses ``u0000``.
```

Comment:
If there are doc changes to be done for options here, let's make sure chaning all. Quick and easy way will be just `grep` and replace.

Reply:
Are you referring to the read part? If so then I left it intentionally there. In the read part there is a test which covers this functionality and works as expected.

CSVSuite.scala:test("SPARK-15585 turn off quotations")

If you mean something else please highlight it allowing me to learn from it.